cosmo_tester/framework/util.py

```diff
@@ -741,3 +741,49 @@ def wait_for_execution(manager, execution, logger):
         )
     logger.info('Execution complete')
     return execution
+
+
+def _get_release_dict_by_name(
```

Comment:
maybe change to '_get_github_release_dict_by_name' just to make it clear what is the use case